prettier.config.js

```diff
@@ -3,7 +3,7 @@ const config = {
 	printWidth: 120,
 	semi: true,
 	singleQuote: true,
-	trailingComma: 'all',
+	trailingComma: 'es5',
```

Comment:
personally, I'm inclined towards "all", curious to hear your opinion for changing this es5

Reply:
no specific reason to use `es5` or `all`. will change to `all` for now.